### ISSUE TITLE
Feature request: Display of state plan for to_dir in multi-state

### ISSUE THREAD

**Issue by fenethtool** [minamijoyo/tfmigrate#138]
Hi, I have always used this software very frequently. It is a great OSS.

# description
```
migration "multi_state" "mv_dir1_dir2" {
  from_dir = "./old"
  to_dir   = "./new"
  force = true
  actions = [
    "mv aws_iam_user.test_user aws_iam_user.test2_user",
  ]
}
```

For example, if we want to perform a state operation across multiple directories as described above, the terraform plan is currently executed only on the directory set on from_dir, and the results are output.
In this case, the source directory can be guaranteed to be normal, but if we want to guarantee the normality of the destination directory, there is no way to check this with the current tfmigrate. Therefore, when executing the `tfmigrate plan`, is it possible to execute the `terraform plan` for the `to_dir` at the same time as the terraform plan for the `from_dir`? Thank you!

**Comment by chrissng** [comment]
Coincidentally this PR would address this issue

https://github.com/minamijoyo/tfmigrate/pull/139

**Comment by minamijoyo** [comment]
Fixed in v0.3.12 via #139. Thanks for @chrissng
